torchvision/datasets/vision.py

```diff
@@ -5,6 +5,17 @@
 
 
 class VisionDataset(data.Dataset):
+    """
+    Base Class For making datasets which are compatible with torchvision.
+    It is necessary to override the ``__getitem__`` and ``__len__`` method.
+
+    Args:
+        root (string): Root directory of dataset.
+        transforms (callable, optional): A function/transform that takes in a PIL image
```

Comment:
Since `transform` is deprecated. I have avoided adding that in `Args`.
Recommended way is only mentioned.

Reply:
we should still mention it and indicate that it's deprecated. Otherwise it may lead to confusion

Reply:
> Since `transform` is deprecated

It is deprecated? Was there a PR for this? And if yes, why is there no warning in case a dataset is instantiated with `transform`?